### PR TITLE
Update authentication

### PR DESCRIPTION
**Pull request by elbayaninho**

### CODE REVIEW

File: backend/backend/urls.py [hunk 1]
```diff
@@ -1,7 +1,13 @@
...
 from django.urls import path, include
+from django.conf import settings
+from django.conf.urls.static import static
  
 urlpatterns = [
     path('admin/', admin.site.urls),
     path('api/', include('core.urls')),
-]
\ No newline at end of file
+    path('api-auth/', include('rest_framework.urls'))
```
Comment (yas228) [inline backend/backend/urls.py:9]: What is the goal of the api-auth route?

**Comment by elbayaninho** [comment]
Login and register



Sent from my Galaxy



-------- Original message --------
From: Yawo Siatitse ***@***.***>
Date: 8/31/23 00:07 (GMT+01:00)
To: JUICEME-INC/juiceme ***@***.***>
Cc: Baïmi Badjoua ***@***.***>, Author ***@***.***>
Subject: Re: [JUICEME-INC/juiceme] Update authentication (PR #2)


@yas228 commented on this pull request.

________________________________

In backend/backend/urls.py<https://github.com/JUICEME-INC/juiceme/pull/2#discussion_r1310927707>:

> \ No newline at end of file
+    path('api-auth/', include('rest_framework.urls'))


What is the goal of the api-auth route?

**Comment by yas228** [comment]
Reply to the comment on github

**Comment by elbayaninho** [comment]
Login and register views

**Comment by yas228** [comment]
> Login and register views

We will use Azure AAD for login so research about that please

**Comment by elbayaninho** [comment]
Ok

**Review by yas228 — APPROVED** [review]

**Pull request merged**